### PR TITLE
Remove code to check the server version

### PR DESCRIPTION
**Pull request by guntbert**
will solve #2439

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt [hunk 1]
```diff
@@ -335,7 +335,7 @@ class ComposeActivity :
                 maximumTootCharacters = instanceData.maxChars
                 charactersReservedPerUrl = instanceData.charactersReservedPerUrl
                 updateVisibleCharactersLeft()
-                binding.composeScheduleButton.visible(instanceData.supportsScheduled)
+                binding.composeScheduleButton.visible(true)
```
Comment (connyduck) [inline app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt:338]: I think this line is entirely unnecessary as the button is visible after layout inflation (if not, change it in the layout)

**Review by connyduck — CHANGES_REQUESTED** [review]
Thanks for the help!

File: app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt (hunk 1 above)
Comment (guntbert) [reply app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt:338]: Ah, I found the button - lets see if it stays visible when I remove the line

Comment (guntbert) [reply app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt]: All looks ok - the button is visible

**Review by connyduck — APPROVED** [review]

**Comment by guntbert** [comment]
@connyduck thank you for the guidance :bowing_man:

**Pull request merged**